src/main/app/case/case.ts

```diff
@@ -35,6 +35,7 @@ export const formFieldsToCaseMapping: Partial<Record<keyof Case, keyof CaseData>
   changedNameHowAnotherWay: 'petitionerNameChangedHowOtherDetails',
   respondentEmailAddress: 'respondentEmailAddress',
   knowPartnersAddress: 'petitionerKnowsRespondentsAddress',
+  claimCosts: 'divorceCostsClaim',
```

Comment:
Not a fan of this variable being named 'divorceCostsClaim' in the api but it's being used in the solicitor journey - maybe we need a task to make the variable names apply for both div and dissolution of cp?

Reply:
there is of cases like this, for example nfdiv 302 uses marriage as one of its options.

Reply:
Yeah I've seen a few, thats why I thought it might be better to change them all in a pr rather than just this field for this ticket